libs/full/parcelset/tests/regressions/CMakeLists.txt

```diff
@@ -29,10 +31,13 @@ foreach(test ${tests})
     FOLDER "Tests/Regressions/Modules/Full/Parcelset"
   )
 
-  # Disable the test due to limited CircleCI resources
-
-  # add_hpx_regression_test( "modules.parcelset" ${test} ${${test}_PARAMETERS}
-  # TIMEOUT 900 )
+  # Disable the test for very_big_parcel due to limited CircleCI resources
+  if(${test} STREQUAL "very_big_parcel")
+    continue()
+  endif()
```

Comment:
Wouldn't it be better to add an exclusion here: https://github.com/STEllAR-GROUP/hpx/blob/master/.circleci/config.yml#L706?